sys/include/net/netopt.h

```diff
@@ -52,24 +52,22 @@ typedef enum {
      */
     NETOPT_IS_CHANNEL_CLR,
     /**
-     * @brief   (various, see below) link layer address
+     * @brief   (byte array, see below) link layer address
      *
-     * Device type   | Type     | Meaning
-     * ------------- | -------- | -----
-     * IEEE 802.15.4 | uint16_t | device short address
-     * Ethernet      | FIXME    | device MAC address??
+     * Device type   | Length | Meaning
+     * ------------- | ------ | -----
+     * IEEE 802.15.4 | 2      | device short address
+     * Ethernet      | 6      | device MAC address
      */
     NETOPT_ADDRESS,
 
     /**
-     * @brief   (various, see below) long link layer address
-     *
-     * Examples for this include the EUI-64 in IEEE 802.15.4
+     * @brief   (byte array, see below) long link layer address
      *
-     * Device type   | Type         | Meaning
-     * ------------- | ------------ | -----
-     * IEEE 802.15.4 | @ref eui64_t | device long address (EUI-64)
-     * Ethernet      | FIXME        | device MAC address??
+     * Device type   | Length   | Meaning
+     * ------------- | -------- | -----
+     * IEEE 802.15.4 | 8        | device long address (EUI-64), @ref eui64_t
+     * Ethernet      | -ENOTSUP | not used
```

Comment:
Just leave it out. Otherwise you would have to add `cc110x`, BLE etc. here as well ;-).